src/cli/helpers.py

```diff
@@ -217,15 +218,27 @@ def record_struct(name, search, search_type, record_links, save, category, subca
     language = "general"
     name = re.sub("[^A-Za-z0-9]+", "", name)
     name = name + "_file_check"
-    save_path = Path("assessments") / save / "evidence.json"
+    if environment == 'production':
+        save_path = Path("assessments/"+save+"/evidence.json")
```

Comment:
Just a nitpick, to concatenate things with a Path object you shouldn't use + use / 
```python
save_path = Path("assessments") / save / "evidence.json"
```

Reply:
Thanks will make the change on the next one / future.